.github/workflows/tests-datapath-verifier.yaml

```
@@ -0,0 +1,237 @@
+name: Cilium Datapath BPF Verifier Complexity
```

Nit: 'Cilium' is kind of implied here. This gets quite long in the job view at the bottom and becomes too wide for the box: ` Cilium Datapath BPF Verifier Complexity / Setup & Test (5.4-main) (pull_request) Successful in 4m`

```suggestion
name: Datapath BPF Complexity
```
I think this should be sufficiently specific.